src/content/operations/cleanup/_index.en.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Uninstalling Submariner"
+date: 2020-12-23T21:25:11+01:00
+weight: 50
+---
+
+To properly uninstall Submariner from a cluster, follow the steps below:
+
+1. Delete Submariner related namespaces
+
+   On each participating cluster, issue the following command:
+
+   ```bash
+   kubectl delete namespace submariner-operator
+   ```
+
+   On the Broker cluster, issue the following command:
+
+   ```bash
+   kubectl delete namespace submariner-k8s-broker
+    ```
+
+2. Delete the Submariner CRDs
+
+   On each participating cluster, issue the following command:
+
+   ```bash
+   for CRD in `kubectl get crds | grep -iE 'submariner|multicluster.x-k8s.io'| awk '{print $1}'`; do kubectl delete crd $CRD; done
+   ```
+
+3. Delete ClusterRoles and ClusterRoleBindings
```

Comment:
```suggestion
3. Delete Submariner's `ClusterRole`s and `ClusterRoleBinding`s
```

https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-inline-code-commands-and-api-objects